pyqtorch/utils.py

```diff
@@ -142,3 +143,59 @@ def density_mat(state: Tensor) -> Tensor:
     state = torch.permute(state, batch_first_perm).reshape(batch_size, 2**n_qubits)
     undo_perm = (1, 2, 0)
     return torch.permute(torch.einsum("bi,bj->bij", (state, state.conj())), undo_perm)
+
+
+def promote_ope(operator: Tensor, target: int, n_qubits: int) -> Tensor:
```

Comment:
This PR is only about `apply_ope_ope()`. The function `promote_ope()` is just here to allow the test. If you want to review `promote_ope()`, please use #155.